app/main.py

```diff
@@ -1,2 +1,60 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [None for _ in range(self.capacity)]
+
+    def resize(self) -> None:
+        self.capacity *= 2
+        self.size = 0
+        self.threshold = int(self.capacity * 2 / 3)
+        new_table = self.hash_table.copy()
+        self.hash_table = [None for _ in range(self.capacity)]
+
+        for element in new_table:
+            if element:
+                self.__setitem__(element[0], element[2])
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
The `Key` is probably `Hashable` not just `Any`. What do you think?

Reply:
right, my bad